xmr-btc/src/tor.rs

```diff
@@ -0,0 +1,104 @@
+use anyhow::{anyhow, bail, Result};
+use lazy_static::lazy_static;
+use std::{
+    future::Future,
+    net::{IpAddr, Ipv4Addr, SocketAddr, SocketAddrV4},
+};
+use tokio::net::TcpStream;
+use torut::{
+    control::{AsyncEvent, AuthenticatedConn, ConnError, UnauthenticatedConn},
+    onion::TorSecretKeyV3,
+};
+
+lazy_static! {
+    /// The default TOR socks5 proxy address, `127.0.0.1:9050`.
+    pub static ref TOR_PROXY_ADDR: SocketAddrV4 = SocketAddrV4::new(Ipv4Addr::LOCALHOST, 9050);
+    /// The default TOR Controller Protocol address, `127.0.0.1:9051`.
+    pub static ref TOR_CP_ADDR: SocketAddr = SocketAddr::V4(SocketAddrV4::new(Ipv4Addr::LOCALHOST, 9051));
+}
```

Comment:
:wrench: These are only ever used as `&str`. Could we initialise them as such using `const`s without needing `lazy_static`? Or is this providing something more?

Reply:
Removed in the last patch.